lib/Twig/Extensions/Extension/Array.php

```diff
@@ -50,3 +52,5 @@ function twig_shuffle_filter($array)
 
     return $array;
 }
+
+class_alias('Twig_Extensions_Extension_Array', 'Twig\Extensions\Extension\ArrayExtension', false);
```

Comment:
Adding the "Extension" suffix is a workaround the reserved "array" keyword.
Do you think we should add the suffix everywhere? (`Array_` instead of `ArrayExtension` would be worse imho)